go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1504,18 +1515,21 @@ func commonFlows(subnets []*net.IPNet, bridge *bridgeConfiguration) ([]string, e
 				fmt.Sprintf("cookie=%s, priority=175, in_port=%s, sctp6, ipv6_src=%s, "+
 					"actions=ct(table=4,zone=%d)",
 					defaultOpenFlowCookie, ofPortPatch, physicalIP.IP, HostMasqCTZone))
-			// We send BFD traffic coming from OVN to outside directly using a higher priority flow
+			if ofPortPhys != "" {
```

Comment:
Should BFD traffic be sent to the host?

Reply:
I don't know. I think the BFD is for iCNI. Do we want users to use iCNI in local gw mode with dummy br-ex?

Reply:
I think the whole egress gw feature wouldn't work.

Reply:
Yes, Neither egress IP nor egress GW will work in such a setup.